project.json

```diff
@@ -2,14 +2,6 @@
   "name": "@code-pushup/cli-source",
   "$schema": "node_modules/nx/schemas/project-schema.json",
   "targets": {
-    "local-registry": {
-      "executor": "@nx/js:verdaccio",
-      "options": {
-        "port": 4873,
-        "config": ".verdaccio/config.yml",
-        "storage": "tmp/local-registry/storage"
-      }
-    },
```

Comment:
This target is moved into [`tools/src/verdaccio/verdaccio.plugin.ts`](https://github.com/code-pushup/cli/pull/798/files#diff-311b3179a39053ba23f1bc3842e27f6b87b112b8082b53a3501bd6a669145c7c)